app/src/main/java/com/example/safehopper/ui/contacts/ContactsFragment.java

```diff
@@ -1,51 +1,70 @@
 package com.example.safehopper.ui.contacts;
 
 import android.os.Bundle;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.TextView;
 
-import androidx.annotation.Nullable;
 import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
-import androidx.lifecycle.Observer;
 import androidx.lifecycle.ViewModelProviders;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.example.safehopper.R;
-import com.example.safehopper.RecyclerViewAdapter;
 
 import java.util.ArrayList;
 
 public class ContactsFragment extends Fragment {
 
+    public static final String TAG = "ContactsFragment";
+
     private ContactsViewModel contactsViewModel;
 
-    private RecyclerView recyclerView;
-    private RecyclerViewAdapter recyclerViewAdapter;
-    ArrayList<EmergencyContact>
+    private ArrayList<String> mNames = new ArrayList<>();
+    private ArrayList<String> mPhoneNumbers = new ArrayList<>();
+    private ArrayList<String> mEmails = new ArrayList<>();
 
     public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         contactsViewModel =
                 ViewModelProviders.of(this).get(ContactsViewModel.class);
         View root = inflater.inflate(R.layout.fragment_contacts, container, false);
 
-        //initRecyclerView()
-        recyclerView = root.findViewById(R.id.recycler_view);
-        RecyclerViewAdapter adapter = new RecyclerViewAdapter(this, this);
+        Log.d(TAG, "onCreateView started.");
+
+        initImageBitMaps();
+
+        RecyclerView recyclerView = root.findViewById(R.id.recycler_view);
+        RecyclerViewAdapter adapter = new RecyclerViewAdapter(mNames, mPhoneNumbers, mEmails, this.getContext());
         recyclerView.setAdapter(adapter);
-        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
-
-        /*final TextView textView = root.findViewById(R.id.text_gallery);
-        contactsViewModel.getText().observe(this, new Observer<String>() {
-            @Override
-            public void onChanged(@Nullable String s) {
-                textView.setText(s);
-            }
-        });*/
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+
         return root;
     }
+
```

Comment:
Please change this function name to a name that relates to what is actually happening here.